### PR TITLE
Fix issue eelsdb qt5.9

### PR DESCRIPTION
**Pull request by ericpre**
- [x] Fix issue with the EELS database for recent version of pyqt5,
- [x] update installation instructions.

### CODE REVIEW

File: hyperspyui/plugins/eelsdb.py [hunk 1]
```diff
@@ -127,19 +128,23 @@ def default(self):
...
-        frame.findFirstElement(".footer").removeFromDocument()
-        for el in frame.findAllElements(".navbar"):
-            el.removeFromDocument()
-        self.view.page().setLinkDelegationPolicy(
-                QtWebEngineWidgets.QWebEnginePage.DelegateAllLinks)
-        self.view.linkClicked.connect(self._on_link)
+        # frame = self.view.page().mainFrame()
+        # frame.findFirstElement(".footer").removeFromDocument()
+        # for el in frame.findAllElements(".navbar"):
+        #     el.removeFromDocument()
```
Comment (vidartf) [inline hyperspyui/plugins/eelsdb.py]: I'd rather old code was deleted if no longer used.

Comment (ericpre) [reply hyperspyui/plugins/eelsdb.py]: True, this is now done.

**Comment by ericpre** [comment]
Downloading spectra is currently not working but this PR allows browsing the database, which is a small incremental progress... ;)

**Comment by ericpre** [comment]
I have tried to update this plugin following the way spyder does it (https://github.com/spyder-ide/spyder/blob/master/spyder/widgets/browser.py) but it may need to some work to get the download to work.

**Pull request merged**